syntax = "proto3";

import "RoomInfo.proto";

message GetRoomListS2C {
    int32 error = 1;
	repeated RoomInfo roomsInfo = 2;
}


